### DISCUSSION TITLE
OICA

### DISCUSSION THREAD

**Discussion by paula-fel** [transport-data/community#11] [Sources]
or the “International Organization of Motor Vehicle Manufacturers”

Includes the following indicators:

- vehicle stock

Geography:

- worldwide

Relevant URLs:

- https://www.oica.net/category/vehicles-in-use/

**Comment by khaeru** [comment]
[v24.2.5 of the `transport-data` Python package](https://docs.transport-data.org/en/latest/whatsnew.html#v24-2-5) added support for certain OICA data flows:
- Sales, vehicle in use: fetch and convert to SDMX.
- Production: fetch only.
